src/uwtools/drivers/driver.py

```diff
@@ -25,7 +25,7 @@ def __init__(self, config_file: str, log: Optional[Logger] = None):
     # Public methods
 
     @abstractmethod
-    def batch_script(self) -> None:
+    def batch_script(self, job_resources):
```

Comment:
We seem to have lost the return type here.

Reply:
There had been an issue in a previous iteration that was failing on not returning correctly, but the proper version should just have a non-None type, thanks